release-notes/opensearch-security.release-notes-1.3.6.0.md

```diff
@@ -0,0 +1,10 @@
+## 2022-10-03 Version 1.3.6.0
+
+### Bug fixes
+* Update Kafka Client to 3.0.2 ([#2129](https://github.com/opensearch-project/security/pull/2129))
+* Scope updateVersion to only build.gradle ([#2121](https://github.com/opensearch-project/security/pull/2121))
+* Staging for version increment automation ([#2029](https://github.com/opensearch-project/security/pull/2029))
+
+### Maintenance
+
+* Updates jackson version to be in sync with core ([#2128](https://github.com/opensearch-project/security/pull/2128))
```

Comment:
Let's change this to `Updates jackson version to 2.13.4

Reply:
I think we should leave it as is. If someone is interested in the version number they can check core.

Reply:
Ideally, we should include the version of the dependency that we update to in a release note on a dependency update. In this case, all impacted plug-ins are going to have something similar to this in their release notes so either should be ok.